tutorials/hyperspectral/Advanced_Spectral_Signatures.Rmd

```diff
@@ -0,0 +1,291 @@
+---
+title: "Advanced Spectral Signatures in R"
+code1: HYPERSPECTRAL/Advanced_Spectral_Signatures.R
+contributors: null
+output:
```

Comment:
I think the "output" in the YAML will need to be deleted - not sure what it would do to the syncing.

Reply:
Sure, that's easy. 'output' deleted.